### DISCUSSION TITLE
How to install on cpanel

### DISCUSSION THREAD

**Discussion by demon071** [john-cms/johncms#139] [Q&A] [answered]
I try to install on cpanel but when the install is successful I can't access the website, I don't know what happened, using version 8.0.0 is still normal, but can't use version 9.0.0 or above up. help me

**Comment by simba77** [comment, marked as answer]
What error is displayed?

**Reply by demon071** [reply]
When I access anonymously, I can access the website, but when I log into the website, This page is working
domain.com is currently unable to handle this request.
HTTP ERROR 500

**Reply by simba77** [reply]
Open the file **config/constants.php**. Find the **const DEBUG** line and set it to true. After that, see what error will be displayed.

**Reply by demon071** [reply]
Fatal error: Uncaught TypeError: long2ip() expects parameter 1 to be int, string given in /home/tiktokvt/public_html/forum/system/src/Casts/Ip.php:31 Stack trace: #0 /home/tiktokvt/public_html/forum/system/src/Casts/Ip.php(31): long2ip('1906437782') #1 /home/tiktokvt/public_html/forum/system/vendor/illuminate/database/Eloquent/Concerns/HasAttributes.php(572): Johncms\Casts\Ip->get(Object(Johncms\Users\User), 'ip', '1906437782', Array) #2 /home/tiktokvt/public_html/forum/system/vendor/illuminate/database/Eloquent/Concerns/HasAttributes.php(550): Illuminate\Database\Eloquent\Model->getClassCastableAttributeValue('ip', '1906437782') #3 /home/tiktokvt/public_html/forum/system/vendor/illuminate/database/Eloquent/Concerns/HasAttributes.php(1443): Illuminate\Database\Eloquent\Model->castAttribute('ip', '1906437782') #4 /home/tiktokvt/public_html/forum/system/vendor/illuminate/database/Eloquent/Concerns/HasAttributes.php(386): Illuminate\Database\Eloquent\Model->transformModelValue('ip', '1906437782') #5 /home/tiktokvt/public_htm in /home/tiktokvt/public_html/forum/system/src/Casts/Ip.php on line 31

**Reply by simba77** [reply]
Your hosting does not use the [mysqlnd](https://www.php.net/manual/en/book.mysqlnd.php) driver. You need to configure it so that it is used.

**Reply by demon071** [reply]
it worked, thanks

**Reply by demon071** [reply]
I have one more question, to put the article on the homepage you have any instructions

**Reply by simba77** [reply]
You can read it here: https://johncms.com/documentation/create_template/
